onnx/defs/experiments/defs.cc

```diff
@@ -307,3 +307,29 @@ OPERATOR_SCHEMA(ATen)
 Experimental allowing ATen operations to be accessed directly from Caffe2
 to allow for quick prototyping when ONNX is missing standard versions of
 and op)DOC");
+
+OPERATOR_SCHEMA(ResizeNearest)
+    .SetSupportLevel(SupportType::EXPERIMENTAL)
+    .NumInputs(1)
+    .NumOutputs(1)
+    .Attr(
+        "width_scale",
+        "The scale along width dimension",
+        AttrType::FLOAT)
+    .Attr(
+        "height_scale",
+        "The scale along height dimension",
+        AttrType::FLOAT)
+    .Input(
+        1,
+        "X",
+        "4D tensor, N x C x H x W")
+    .Output(
+        1,
+        "Y",
+        "4D tensor after resizing, N x C x H x W")
+    .SetDoc(R"DOC(
+Resize the width and height dimensions:
```

Comment:
Maybe add some more description? It's not clear what values of the new pixels in the output.

Reply:
Will add more description, and an example here.